index.html

```diff
@@ -1,12 +1,82 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/boxicons@latest/css/boxicons.min.css" />
+    <link rel="stylesheet" href="https://algernui.netlify.app/components/main.css" />
+
+    <link rel="stylesheet" href="style.css">
+    <title>Algern Quiz</title>
 </head>
+
 <body>
-    
+    <header class="home-page-header relative">
+        <nav class="navbar">
+            <a href="#">
+                <h2 class="logo">Algern Quiz</h2>
+            </a>
+            <a href="#">
+                <i class='bx bxs-user icon'></i>
+            </a>
+        </nav>
+
+        <section class="hero">
+            <div class="intro">
+                <h1 class="bold-text">Do you like trivia nights?</h1>
+                <p class="lead">We've got all the quizzes you love to binge! Come on in and hunker down for the long
+                    haul.</p>
+            </div>
+            <div>
+                <img class="hero-img" src="./assets/casual-life-3d-girl-and-boy-sitting-with-tablet.png" alt="girl and">
+            </div>
+        </section>
+    </header>
+
+    <main class="categories">
+        <h3 class="text-center bold-text underline">Categories</h3>
+        <section class="categories-container">
+
+            <article class="category">
+                <a href="#">
+                    <img class="category-img" src="./assets/holding-al-pacino-best-and-worst-film-moments.jpg" alt="">
+                    <div class="desc">
+                        <h4 class="card-title">Famous hollywood characters</h4>
+                    </div>
+                </a>
+            </article>
+
+            <article class="category">
+                <a href="#">
+                    <img class="category-img" src="./assets/holding-al-pacino-best-and-worst-film-moments.jpg" alt="">
+                    <div class="desc">
+                        <h4 class="card-title">Famous hollywood characters</h4>
+                    </div>
+                </a>
+            </article>
+
+            <article class="category">
+                <a href="#">
+                    <img class="category-img" src="./assets/holding-al-pacino-best-and-worst-film-moments.jpg" alt="">
+                    <div class="desc">
```

Comment:
you can give values to alt attributes.

Reply:
Oh yea, totally forgot about them. thank you!